packages/@coorpacademy-components/src/atom/link/index.js

```diff
@@ -72,13 +73,20 @@ const Link = (props, legacyContext) => {
     [hoverColor, hovered, skinHover]
   );
 
+  const _href = useMemo(() => {
+    if (disabled) return '#';
```

Comment:
Maybe we can merge this and the line below into `if (disabled || !href) return;`?

Reply:
yes, I agree, I changed that